js_modules/dagster-ui/packages/ui-core/src/asset-graph/CollapsedGroupNode.tsx

```diff
@@ -0,0 +1,115 @@
+import {Box, Colors, FontFamily, Icon} from '@dagster-io/ui-components';
+import React from 'react';
+import styled from 'styled-components';
+
+import {withMiddleTruncation} from '../app/Util';
+
+import {AssetDescription, NameTooltipCSS} from './AssetNode';
+import {GroupLayout} from './layout';
+
+export const GroupNodeNameAndRepo = ({group, minimal}: {minimal: boolean; group: GroupLayout}) => {
+  const name = `${group.groupName} `;
+  const location = `${group.repositoryName}@${group.repositoryLocationName}`;
+
+  if (minimal) {
+    return (
+      <Box style={{flex: 1, fontFamily: FontFamily.monospace}}>
+        <div
+          data-tooltip={name}
+          data-tooltip-style={GroupNameTooltipStyle}
+          style={{fontSize: 30, fontWeight: 600, lineHeight: '30px'}}
+        >
+          {withMiddleTruncation(name, {maxLength: 14})}
+        </div>
+        <div style={{fontSize: 20}}>{withMiddleTruncation(location, {maxLength: 21})}</div>
+      </Box>
+    );
+  }
+  return (
+    <Box style={{flex: 1, fontFamily: FontFamily.monospace}}>
+      <Box flex={{direction: 'row', gap: 4}}>
+        <div
+          data-tooltip={name}
+          data-tooltip-style={GroupNameTooltipStyle}
+          style={{fontSize: 20, fontWeight: 600, lineHeight: '1.1em'}}
+        >
+          {withMiddleTruncation(name, {maxLength: 22})}
+        </div>
+      </Box>
+      <Box style={{lineHeight: '1em'}}>{withMiddleTruncation(location, {maxLength: 31})}</Box>
+    </Box>
+  );
+};
+
+export const CollapsedGroupNode = ({
+  group,
+  minimal,
+  onExpand,
+}: {
+  minimal: boolean;
+  onExpand: () => void;
+  group: GroupLayout & {assetCount: number};
+}) => {
+  return (
+    <CollapsedGroupNodeContainer
```

Comment:
Should we use a button for accessibility?

Reply:
I think we may be able to, but let's try moving all these to button (Asset Node, Group Node etc) in one go in a follow-up, I think allowing these elements to have focus could mean that we want to remove the concept of a focused / "last clicked item" from the React state so they don't have to be kept in sync?